editor/src/components/inspector/common/css-utils.ts

```diff
@@ -951,23 +952,21 @@ export const disabledFunctionName = UtopiaUtils.disabled.name
 
 export function printBoxShadow(boxShadows: CSSBoxShadows): JSXAttributeValue<string> {
   const indexOfLastEnabledLayer = findLastIndex(isLayerEnabled, boxShadows)
-  return jsxAttributeValue(
-    [...boxShadows]
+  return jsxAttributeValue([...boxShadows]
       .map((boxShadow, i) => {
-        const comma = indexOfLastEnabledLayer > i && boxShadow.enabled
-        const { inset, offsetX, offsetY, blurRadius, spreadRadius, color } = boxShadow
-        const parts = Utils.stripNulls([
+      const comma = indexOfLastEnabledLayer > i && boxShadow.enabled
+      const {inset, offsetX, offsetY, blurRadius, spreadRadius, color} = boxShadow
```

Comment:
this is weird! why is the formatting different here? shouldn't Prettier put a space between the `{` and `i`?

Reply:
Your guess is as good as mine, answers on a postcard please.

Reply:
It seems like something strange has happened with either prettier or eslint, because a bunch of indenting has changed in this file too

Reply:
maybe the Jetbrains auto refactor re-printed the entire file, and then it skipped prettier?

Reply:
We run prettier from a commit hook though right?

Reply:
> We run prettier from a commit hook though right?

AFAIK yes, unless that broke recently?

Reply:
Yeah we still run it in a commit hook, and the config doesn't appear to have changed at all

Reply:
however, I think many of us have their IDE also run prettier, so we might not see it if it silently breaks

Reply:
I believe if it failed to run it would fail the commit